app/src/main/java/com/example/mycalculator/mvp/Operand.java

```diff
@@ -0,0 +1,48 @@
+package com.example.mycalculator.mvp;
+
+import static com.example.mycalculator.utils.NumbersUtils.DOUBLE_ZERO_NUMBER;
+import static com.example.mycalculator.utils.StringUtils.EMPTY_STRING;
+
+public class Operand {
+
+    public String sign = EMPTY_STRING;
+    public String value = EMPTY_STRING;
+
+    private String getSign() {
+        return sign;
+    }
+
+    public void eraseOperands() {
+        sign = EMPTY_STRING;
+        value = EMPTY_STRING;
+    }
+
+    public Double getValue() {
+        if (value == EMPTY_STRING) {
+            return DOUBLE_ZERO_NUMBER;
+        } else {
+            return Double.parseDouble(sign + value);
+        }
+    }
+
+    public void setSign(String sign) {
+        this.sign = sign;
+    }
+
+    public void setValue(String value) {
+        this.value = value;
+    }
+
+    public Boolean isEmpty() {
+        return value.isEmpty();
+    }
+
+    public String getFullResult() {
+        return null;
```

Comment:
this method it used? and why you return a null?

Reply:
You were right, I was not using those methods, so they were deleted